docs/source/scheduled_tasks.rst

```diff
@@ -0,0 +1,31 @@
+.. _scheduled-tasks:
+
+Scheduled Tasks
+===================
+
+To create a new experiment-specific background tasks, you can register classmethods
+or functions on your experiment class using the :attr:`~dallinger.experiment.scheduled_task`
+decorator::
+
+    from dallinger import experiment
+
+    class MyCustomExperiment(experiment.Experiment):
+        ...
+
+        @scheduled_task("interval", minutes=15)
```

Comment:
Should this be `@experiment.scheduled_task("interval", minutes=15)` ?

Reply:
Indeed it should.